src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring.mdx

```diff
@@ -0,0 +1,498 @@
+---
+title: Configure control plane monitoring
```

Comment:
Incredible work with this doc !!

Reply:
A bunch of the paragraphs were copied or inspired by your work on the internal confluence page!